dev/merge_spark_pr.py

```diff
@@ -72,6 +72,13 @@
 BRANCH_PREFIX = "PR_TOOL"
 
 
+def print_error(msg):
+    print("\033[91m%s\033[0m" % msg)
```

Comment:
Just a question. Does this color code work in all shell and OS environments?

Reply:
https://en.wikipedia.org/wiki/ANSI_escape_code#Colors

Not all terminals and operating systems, but most of them support ANSI colors